src/auto_code_demogs.py

```diff
@@ -50,6 +50,36 @@ def auto_code_demogs(cls, user, data, phone_uuid_table, coda_output_dir):
                 )
             td.append_data({"operator_coded": label.to_dict()}, Metadata(user, Metadata.get_call_location(), time.time()))
 
+        # Subsample messages for export to coda
+        subsample_data = MessageFilters.subsample_messages(data)
+
+        # Output single-scheme subsample answers to coda for manual verification + coding
+        IOUtils.ensure_dirs_exist(coda_output_dir)
+        for plan in PipelineConfiguration.DEMOG_CODING_PLANS:
+            if plan.raw_field == "location_raw":
```

Comment:
Why are we skipping location?

Reply:
This is handled separately by the block of code 97 -109